docs/getting-started/index.md

```diff
@@ -7,14 +7,14 @@ navigation_weight: 1
 # Getting Started
 
 If you are completely new to service workers and sw-helpers,
-the easiest place to start with `sw-cli`.
+the easiest place to start is `sw-cli`.
 
-This is a command line tool that can build a service
-worker that will precache the assets for your site and make it work offline.
+This is a command line tool that will build a service
+worker to precache assets for your site and make it work offline.
 
 [Learn More About sw-cli Here](../reference-docs/stable/latest/module-sw-cli.html#main)
 
-If you'd like to make use of the functionality provided by these libraries
-in your own service worker than `sw-lib` would be the best option for you.
+If you'd like to make use of the sw-helpers libraries, and you already have your own 
```

Comment:
This is in Sam's fork. Either he needs to do it, or I'll need to wait for the merge.